lib/ui/view/start_dm/start_dm_viewmodel.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
```

Comment:
remove material import, use [FormView](https://www.filledstacks.com/post/building-flutter-login-and-sign-up-forms/) to use TEController without importing material

Reply:
I don't understand this, and seeing as there is only 15 minutes. Please accept this. I will change it later